lib/init.coffee

```diff
@@ -11,4 +15,44 @@ module.exports =
         'Available codes: https://pypi.python.org/pypi/pep257#error-codes')
 
   activate: ->
-    console.log 'activate linter-pep257'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-pep257.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @subscriptions.add atom.config.observe 'linter-pep257.ignoreCodes',
+      (ignoreCodes) =>
+        @ignoreCodes = ignoreCodes
+
+  deactivate: ->
+    @subscriptions.dispose()
+
+  lintPath: (params) ->
+    return helpers.exec(@executablePath, params, {stream: 'stderr'})
+
+  parseMessages: (output) ->
+    # parse lint output
+    messages = helpers.parse(output, "^(?<file>.+):(?<line>\\d+)\\s+(?<message>.+)")
+    output = output.split(/\r?\n/)
+    messages.forEach (msg, i) ->
+      msg.type = "Info"
+      text = (output[i+i+1] || "").trim()  # get the second line of each message
+      if text
+        msg.text = text + " (" + msg.text + ")"
```

Comment:
I found that the info in `text` is more descriptive than the info in `msg.text` so in order to have that information show up right off the bat, I've combined them into `msg.text`.